web/styles/search.css

```diff
@@ -137,15 +137,22 @@
         letter-spacing: normal;
         padding-left: 0;
     }
-}
 
-.typeahead-menu li a {
```

Comment:
This is better scoped within the #searchbox classname that wraps the rest of this file